GTG/gtk/data/task_editor.ui

```diff
@@ -61,7 +61,7 @@
             <property name="receives_default">False</property>
             <property name="tooltip_text" translatable="yes">Permanently remove this task</property>
             <property name="action_name">app.editor.delete</property>
-            <property name="text" translatable="yes">Delete Task</property>
+            <property name="text" translatable="yes">Delete</property>
```

Comment:
I don't think we should make this shorter for translating purposes. Only if it makes sense for the dialog conceptually.